samples/containers/app-poetry/src/main/java/com/squareup/sample/poetryapp/PoetryActivity.kt

```diff
@@ -42,7 +42,7 @@ class PoetryActivity : AppCompatActivity() {
 
 class PoetryModel(savedState: SavedStateHandle) : ViewModel() {
   val renderings: Flow<Screen> by lazy {
-    renderWorkflowIn(
+    com.squareup.workflow1.android.renderWorkflowIn(
```

Comment:
Nit: You're already statically importing this, I think the FQN is just an IDE bug? Search-and-replace to fix?

Reply:
Done.